claragenomics/dl4atac/metrics.py

```diff
@@ -16,82 +16,72 @@
 from scipy.stats import rankdata
 
 class Metric(object):
-    """Metrics class"""
+    """Base metrics class."""
 
     def __init__(self):
         """Initialize class."""
-
         self.reset()
 
     def reset(self):
-        """Reset class."""
-
+        """Reset value of metric."""
         self.val = torch.tensor(0.).cuda() if torch.cuda.is_available() else torch.tensor(0.)
 
     def get(self):
-        """Get class."""
-
+        """Get metrics value."""
         return self.val
 
     def __str__(self):
-        """Return string."""
-
+        """Serialize metric into a string."""
         return "{}:{:7.4f}".format(type(self).__name__.lower(), self.val)
 
     def __call__(self, x, y):
-        """
-        Callable object.
+        """Call the metric class as a function.
 
         Args:
-            x: 
-            y: 
-    
+            x: Value 1
+            y: Value 2
+
         Raise:
             Not implemented error
-    
         """
         raise NotImplementedError("Abstract class method called.")
 
     def better_than(self, metric):
-        """
-        Callable object.
+        """Check if current metric is better than given metric.
 
         Args:
-            metric: 
-    
-        Raise:
-            Not implemented error
-    
+            metric: Another instance of metric.
+
+        Return:
+            True : if given metric is greater than current metric.
+            False : if given metric is lesser than current metric.
+            Not implemented error if not implemented.
         """
         raise NotImplementedError("Abstract class method called.")
 
     @staticmethod
     def convert_to_tensor(x):
-        """
-        Convert to tensor.
+        """Convert value to torch tensor.
 
         Args:
-            x: 
-    
+            x: Input value.
+
         Return:
-            x:
-    
+            x: Torch tensor encoded value.
         """
         if not isinstance(x, torch.Tensor):
             x = torch.from_numpy(x)
         return x
 
     @staticmethod
     def convert_to_numpy(x):
-        """
-        Convert to numpy.
+        """Convert value to numpy.
 
         Args:
-            x: 
-    
+            x: Input value.
```

Comment:
Input value should be a list or scalar ?

Reply:
Updated